README.md

```diff
@@ -27,7 +27,7 @@ as WebRTC multiplexes traffic on a single socket but PRs are welcomed
 ```elixir
 def deps do
   [
-    {:ex_ice, "~> 0.3.0"}
+    {:ex_ice, "~> 0.3.1"}
```

Comment:
New feature in patch, hmmmm...
I know it doesn't change existing API, but still...

Reply:
I can do this in minor